src/use/router/index.ts

```diff
@@ -122,6 +123,15 @@ const router = createRouter({
                 roles: ["Lecturer", "Admin"],
             },
         },
+        {
+            path: "/createExamReg",
```

Comment:
```suggestion
            path: "/create-exam-reg",
```
Unfortunately, our URLs for the different components have different cases. We are planning to refactor them, s.t. all path are kebab-case. So we should just use kebab-case for the new features, too :)